action.yml

```diff
@@ -9,6 +9,13 @@ inputs:
     download-repository:
         description: "Remote repository in Artifactory pointing to 'https://releases.jfrog.io/artifactory/jfrog-cli'. Use this parameter in case you don't have an Internet access."
         required: false
+    aud:
```

Comment:
```suggestion
    openid-audience:
```

Reply:
Are you sure about it? I tried to follow the OIDC docs and the Claims in the JWT

Reply:
It was only to make it clearer that this is related to openid and that it is the audience field. Just "aud" is confusing.
Let's also ensure to change the parameter name in the typescript/javascript code.